apps/src/aiComponentLibrary/chatMessage/ChatMessage.tsx

```diff
@@ -71,12 +73,23 @@ const ChatMessage: React.FunctionComponent<ChatMessageProps> = ({
         <div className={moduleStyles.messageWithChildren}>
           <div className={moduleStyles[`container-${role}`]}>
             {role === Role.ASSISTANT && (
-              <div className={moduleStyles.botIconContainer}>
-                <img
-                  src={aiBotOutlineIcon}
-                  alt={commonI18n.aiChatBotIconAlt()}
-                  className={moduleStyles.botIcon}
-                />
+              <div
+                className={classNames(
+                  isTA && moduleStyles.botIconContainerWithOverlay
+                )}
+              >
+                <div className={classNames(moduleStyles.botIconContainer)}>
+                  <img
+                    src={aiBotOutlineIcon}
+                    alt={commonI18n.aiChatBotIconAlt()}
+                    className={moduleStyles.botIcon}
+                  />
+                </div>
+                {isTA && (
+                  <div className={moduleStyles.botOverlay}>
+                    <span>{'TA'}</span>
```

Comment:
Probably should add `TA` to the i18n strings, no?

That gets me wondering whether other languages have a common acronym similar to "TA" that will fit into this small space we've allocated.

Reply:
I'm not 100% sure "TA" is meant to be translated or if it's more language-independent branding? The FAB button's TA tag is actually an image with TA on it, which makes me think it's not meant to be translated

Reply:
From Tess on slack- TA is meant to be branding and should not be translated